config/overlays/odh/params.env

```diff
@@ -2,3 +2,5 @@ odh-mm-rest-proxy=quay.io/opendatahub/rest-proxy:fast
 odh-modelmesh-runtime-adapter=quay.io/opendatahub/modelmesh-runtime-adapter:fast
 odh-modelmesh=quay.io/opendatahub/modelmesh:fast
 odh-modelmesh-controller=quay.io/opendatahub/modelmesh-controller:fast
+etcd=registry.redhat.io/openshift4/ose-etcd-rhel9@sha256:ea7545b79599f3868d442fdffdfe9b12a02a4b56ac155f02c0fac4720d475796
+ose-cli=registry.redhat.io/openshift4/ose-cli-rhel9@sha256:a11ea4a722fadc2cdf532675dd409d384342bea15559aec4b10c2b12ab6d3041
```

Comment:
It might be better if these defaults are specified using tags rather than digests, and then leaving the pinning to specific digests to the overrides that the operator will pass down. WDYT?

Reply:
Ok, that makes sense. I kept the SHA aligned with the actual behavior. 
As for the tag, I noticed that the latest one is currently set to v4.15 instead of v4.19. So, I'd use v4.19 as the tag for both.